### ISSUE TITLE
Pagination with large number ?

### ISSUE THREAD

**Issue by mylastof** [Dogfalo/materialize#1112]
<pre>&lt;ul class="pagination"&gt;
	&lt;li class="waves-effect"&gt;&lt;a href="#!"&gt;&lt;i class="mdi-navigation-chevron-left"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;
	&lt;li class="active"&gt;&lt;a href="#!"&gt;1501&lt;/a&gt;&lt;/li&gt;
	&lt;li class="waves-effect"&gt;&lt;a href="#!"&gt;1502&lt;/a&gt;&lt;/li&gt;
	&lt;li class="waves-effect"&gt;&lt;a href="#!"&gt;1503&lt;/a&gt;&lt;/li&gt;
	&lt;li class="waves-effect"&gt;&lt;a href="#!"&gt;1504&lt;/a&gt;&lt;/li&gt;
	&lt;li class="waves-effect"&gt;&lt;a href="#!"&gt;1505&lt;/a&gt;&lt;/li&gt;
	&lt;li class="waves-effect"&gt;&lt;a href="#!"&gt;&lt;i class="mdi-navigation-chevron-right"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;
&lt;/ul&gt;</pre>
Try using large numbers in pagination, but does not display the numbers intended and only display 3 digits only.
demo : https://jsfiddle.net/mylastof/n1eap8c3/

<i>Request: <strong>Next/Prev pagination</strong> and alignment to left, center. right</i>

**Comment by tmd-uk** [comment]
Check your jsfiddle, I have just updated it for you. You can override your css file with the new values for pagination.

**Comment by mylastof** [comment]
thanks @tmd-uk ,,
but iI've update CSS for small number
```css
.pagination li{
    width: auto;
    height: 35px;
    padding: 5px 15px;
}
```

**Comment by Dogfalo** [comment]
Fixed in c36c3bbd2a474ac096921cc2408ae1f533a94823
